private/bufpkg/bufcheck/buflint/testdata/validate_rules_types_match/buf/validate/expression.proto

```diff
@@ -0,0 +1,92 @@
+// Copyright 2023 Buf Technologies, Inc.
```

Comment:
As opposed to needing to vendor this in every rule, in https://github.com/bufbuild/buf/pull/2033 I proposed we add a new helper function `testLintWithValidate`, that will sideload the protovalidate files into the bucket for the module. You'll likely then end up storing the validate files just once, and outside the specific sub-directory for the given rule, but for extra credit you could curl and cache the module, similar to what we do for googleapis in this repository - search around this repository for googleapis to see prior art.